js/app/dashboard/components/TaskList.js

```diff
@@ -1,42 +1,89 @@
 import React from 'react'
-import { findDOMNode } from 'react-dom'
 import _ from 'lodash'
 import Task from './Task'
 import TaskGroup from './TaskGroup'
 import { connect } from 'react-redux'
-
+import { setTaskListGroupMode } from '../store/actions'
 
 class TaskList extends React.Component {
```

Comment:
Maybe we should rename this to `UnassignedTaskList` ?

Reply:
The ["naming things is hard" label](https://github.com/coopcycle/coopcycle-web/labels/naming%20things%20is%20hard) is here for that 😍  

I created #223